subs.mk

```diff
@@ -2,7 +2,7 @@ KEYSET ?= snakeoil
 DOCKER_BASE ?= docker://
 UBUNTU_MIRROR ?= http://archive.ubuntu.com/ubuntu
 KEYSET_D ?= $(HOME)/.local/share/machine/trust/keys/$(KEYSET)
-MOSCTL_BINARY ?= https://github.com/project-machine/mos/releases/download/v0.0.14/mosctl
+MOSCTL_BINARY ?= /home/serge/src/project-machine/mos/mosctl
```

Comment:
whoops!

Reply:
Oh no - I told myself not to add that file to the index but i guess i ended up git add -a'ing after all.